source/documentation/onboarding.html.md

```diff
@@ -1,26 +1,144 @@
 ---
-author: jruzicka
+author: jruzicka,chandankumar
 title: Getting started at RDO
 ---
 
-## Packager onboarding guide
+# Let's Contribute to RDO Project
 
-Welcome on board!. As new packager there are some things that you must do to
+Welcome on board!. As new contributor there are some things that you must do to
 get ready:
 
-1. Join our [mailing list](/community/mailing-lists/)
-2. You can introduce yourself sending a mail to the mailing list. Tell us what are your
-interests and how you will participate.
-3. Joining our IRC channel #rdo in freenode is a good way to get in touch with other packagers.
-4. Create your account in [RDO SoftwareFactory](https://review.rdoproject.org/auth/login).
-You need a [github](https://github.com) account which is used to log in.
-5. Read our [RDO packaging introduction](/documentation/intro-packaging/). It provides some
-interesting information for newcomers.
-6. Install [rdopkg](/documentation/intro-packaging/#rdopkg) tool, you'll need it!
-7. You can start getting familiar with RDO by reviewing patches in [RDO Gerrit instance](https://review.rdoproject.org/r)
+## Let's get prerequisites ready.
+* [Join the RDO Mailing List](#join-mailing-list)
+* [Join IRC channel #rdo on Freenode server](#join-irc-channel)
+* [Create Github account](#github-account)
+* [Sign to review.rdoproject.org](#sign-in)
+* [Add SSH key to review.rdoproject.org](#add-ssh)
+* [It would be good to have Red Hat Bugzilla account.](#bugzilla-account)
+* [Configure your git and git review.](#git-review)
+* [Install rdopkg tool](#rdopkg)
+
+## [What's Next!](#what-next)
+## [Reviewing RDO Patches](#review-rdo)
```

Comment:
I'd propose following organization of this part:

## What's next! (no link here)
* Do your first contribution
* Reviewing RDO Patches
* Become a RDO package maintainer

These three with links to below sections as done with "Let's get prerequiesites ready"